### PR TITLE
Budget Universal Template File

### LINKED ISSUE #284: Universal Budget Template File

### PR DESCRIPTION
**Pull request by glassblowerscat**
Closes #284 

- Adds the `FileActionsContext` to the Project Budget Scene, along with instances of the cards for adding and viewing/editing Defined Files
- Refactors `ProjectBudget` to give the Budget Records Table its own component, for tidiness

### CODE REVIEW

**Review by CarsonF — COMMENTED** [review]
The table elevation looks weird being next to a different one, no?

Screen Shot 2020-10-08 at 12 19 52 PM

**Comment by glassblowerscat** [comment]
> The table elevation looks weird being next to a different one, no?

Yeah, it kinda does. Which one do you want to change? The drop shadow is maybe more our standard look

File: src/scenes/Projects/Budget/ProjectBudget.graphql [hunk 1]
```diff
@@ -41,6 +41,34 @@ query ProjectBudget($id: ID!) {
...
+    input: {
+      budget: { id: $id, universalTemplateFile: $universalTemplateFile }
+    }
+  ) {
+    budget {
+      id
+      universalTemplateFile {
+        value {
+          id
+          name
```
[the remark this answers is not in the document]
Comment (glassblowerscat) [reply src/scenes/Projects/Budget/ProjectBudget.graphql:70]: Well, we don't really neeeeeeeed all that info, so I was trying not to over-query. Do you want it just for consistency?

**Comment by CarsonF** [comment]
Yeah let's change the records table

**Comment by glassblowerscat** [comment]
> Yeah let's change the records table

Okay, fixed

File: src/components/Table/Table.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ const Container = withStyles({
     },
   },
   // This is the default Container, only styles above have changed.
```
Comment (CarsonF) [inline src/components/Table/Table.tsx:48]: Will you remove this comment now that it's incorrect?

Comment (glassblowerscat) [reply src/components/Table/Table.tsx]: Well, I'll update it 😉

**Review by CarsonF — APPROVED** [review]
Thanks @glassblowerscat!

**Pull request merged**